capsules/Cargo.toml

```diff
@@ -8,3 +8,4 @@ edition = "2021"
 kernel = { path = "../kernel" }
 enum_primitive = { path = "../libraries/enum_primitive" }
 tickv = { path = "../libraries/tickv" }
+cortexm = { path = "../arch/cortex-m" }
```

Comment:
We cannot make capsules dependent on something chip specific.

Instead, we have to use a HIL to abstract HW from capsules.

Reply:
Ok, I'll take a look and amend the pr